### DISCUSSION TITLE
accurate degradation of 90s broadcast inspired graphics using ntsc-rs

### DISCUSSION THREAD

**Discussion by dawnch0rus** [ntsc-rs/ntsc-rs#60] [Show and tell]
one of my favorite usecases that i've found for ntsc-rs is using it to degrade clean digital graphics to a quality that they would've had if they went thru the actual analog processing chain.

up until the late 90s, the process for creating television graphics was very analog. the technology was for sure improving with more complex graphics possible, but up until fully-digital pipelines became a thing there would always be a layer of analog noise over even the cleanest signals. even y/c (aka s-video) with all its improvements never really shook the dominance of good ol cvbs.

for most of the 80s up into the late 90s, the workflow for creating tv graphics looked like this:
-graphic is created, most likely inside a quantel paintbox.
-over cvbs, a vtr records this graphic to videotape.
-the graphic on videotape is fed into a dve to make it move, scale, etc.

if your goal is to recreate 80s/90s tv visuals, it's easy enough to make a graphic and do dve-like things to it in modern software. but what about the subtle analog distortions that occur along the way? this is something that i've found ntsc-rs to be extremely good at simulating.

by daisy-chaining presets that replicate the subtle loss of a clean composite signal and the lossier 1" videotape, i've fairly convincingly recreated what would've went in to the dve - exclusive of needing a massive vtr and a vintage graphic system worth thousands of dollars. here's some screenshots of that in use on my own graphics!

![vlcsnap-2023-12-27-05h59m33s157_ntsc2_ntsc](https://github.com/valadaptive/ntsc-rs/assets/154479340/1305e4ee-2aa8-4b08-b424-9a98d75bb1c6)
![infomercial_screen2_ntsc_ntsc](https://github.com/valadaptive/ntsc-rs/assets/154479340/82f8f250-0a76-4556-849f-8ca503951f47)
![ntsc-rs_ntsc_ntsc](https://github.com/valadaptive/ntsc-rs/assets/154479340/4ca985a7-8cce-4570-85ff-3554c73bbbcf)
![philipscard_ntsc_ntsc_ntsc](https://github.com/valadaptive/ntsc-rs/assets/154479340/38825a48-cd93-4eac-8466-7e477bfdaddd)
![paint_box_graphic_ntsc_ntsc](https://github.com/valadaptive/ntsc-rs/assets/154479340/aaf99d64-2cbb-449c-b942-ebd43f8b133d)
![nvplogoPaintbox_ntsc_ntsc](https://github.com/valadaptive/ntsc-rs/assets/154479340/b5c28e35-b481-47db-b0be-15a0b2a5788b)

if anyone is interested in recreating these results yourself, feel free to ask me for the presets i used. thank you for making this wonderful little effect, i'm of the opinion that it's THE best general purpose analog video simulator that exists. i can't wait to see where else it might go in the future.

**Comment by VexorionReal** [comment]
@nlolnlolnlolnlo Can I have the presets, please?

**Comment by raccoonleader69** [comment]
hey! just made an account to enquire about this preset - any chance of sharing it?

**Comment by dawnch0rus** [comment]
shoot i totally forgot to post these. here they are
[ntsc-rs_1inchapprox_v2_7.json](https://github.com/valadaptive/ntsc-rs/files/15058099/ntsc-rs_1inchapprox_v2_7.json)
[ntsc-rs_cvbs4.5.json](https://github.com/valadaptive/ntsc-rs/files/15058100/ntsc-rs_cvbs4.5.json)

i havent checked these with the new version but they should work fine

**Comment by Degamisu** [comment]
Where did you get the images? They look really cool!

**Reply by dawnch0rus** [reply]
i created all the graphics besides the test card

**Reply by Degamisu** [reply]
Nice. What program? AE?

**Reply by dawnch0rus** [reply]
photoshop

**Reply by smfthe1199** [reply]
Speaking of the image @nlolnlolnlolnlo , What font used in these graphics?

**Reply by dawnch0rus** [reply]
various fonts, mostly helvetica though

**Reply by smfthe1199** [reply]
Thanks for this!

**Comment by smfthe1199** [comment]
Thank you!

**Comment by ntsc-rs-fan** [comment]
This is nice. The effect looks like the original masters of a program that were given to Columbia Pictures Television,  etc.
